File: packages/android/src/main/kotlin/gov/civui/components/CivActionChip.kt
// CivUI — CivActionChip for Jetpack Compose
// Fire-and-forget rounded chip button. Mirrors civ-action-chip.
//
// Placeholder body — see .claude/rules/audit-debt.md → "Native platform implementation pass".

package gov.civui.components

import androidx.compose.foundation.layout.Box
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier

@Composable
fun CivActionChip(
    label: String = "",
    value: String = "",
    disabled: Boolean = false,
    spacing: String = "default",
    iconStart: String = "",
    iconEnd: String = "",
    count: Int = 0,
    modifier: Modifier = Modifier,
    onClick: ((String) -> Unit)? = null,
    onAnalytics: ((event: String, data: Map<String, Any>?) -> Unit)? = null,
) {
    Box(modifier = modifier) {
        // Placeholder — see audit-debt.md for native implementation status.
    }
}

File: packages/ios/Sources/CivUI/CivActionChip.swift
// CivUI — CivActionChip for SwiftUI
// Fire-and-forget rounded chip button. Mirrors civ-action-chip.
//
// Placeholder body — the prop surface satisfies schema parity. See
// .claude/rules/audit-debt.md → "Native platform implementation pass".

import SwiftUI

public struct CivActionChip: View {
    /// Visible label.
    public var label: String

    /// Identifier passed in the click event detail.
    public var value: String

    /// Disabled state.
    public var disabled: Bool

    /// Padding density. `default` | `sm`.
    public var spacing: String

    /// Leading icon name.
    public var iconStart: String

    /// Trailing icon name (rendered after the label and count).
    public var iconEnd: String

    /// Optional count rendered as " (N)" after the label. `nil` = no count.
    public var count: Int?

    /// Click handler.
    public var onClick: ((String) -> Void)?

    /// Analytics tracker (parallels civ-analytics).
    public var onAnalytics: ((String, [String: Any]?) -> Void)?

    public init(
        label: String = "",
        value: String = "",
        disabled: Bool = false,
        spacing: String = "default",
        iconStart: String = "",
        iconEnd: String = "",
        count: Int? = nil,
        onClick: ((String) -> Void)? = nil,
        onAnalytics: ((String, [String: Any]?) -> Void)? = nil
    ) {
        self.label = label
        self.value = value
        self.disabled = disabled
        self.spacing = spacing
        self.iconStart = iconStart
        self.iconEnd = iconEnd
        self.count = count
        self.onClick = onClick
        self.onAnalytics = onAnalytics
    }

    public var body: some View {
        EmptyView()
    }
}
